src/executables/csql_launcher.c

```diff
@@ -434,7 +434,8 @@ main (int argc, char *argv[])
          }
          else if (explicit_single_line && csql_arg.single_line_execution == false)
          {
-         /* Don't allow both at once. */
+       */
```

Comment:
The compiler does not like having the ```/*``` inside a multi line comment so i decided to split them into 2 but keep the headline-like formatting for ```/* Don't allow both at once. */```

Reply:
I think the best way here is to keep only one comment and remove the extra code. It looks bad anyway.

```c
  else if (!csql_arg.sa_mode && csql_arg.skip_vacuum)
    {
      /* Don't allow to skip vacuum on CS mode */
      goto print_usage;
    }
```

Also, even though the change is small, the change to get rid of tmpnam warning will be more complicated and this may be missed. I propose either of two ways forward:

- Move this to a PR with fewer changes like #2650 . Update the PR title & description with both warnings.
- Create a new issue/PR for this change.

I guess first would be easiest :)

Reply:
i will pick the first then :)